site/index.html

```diff
@@ -21,7 +21,7 @@
           I build for the web and love exploring new ways of doing things. So far I love to work with <a href="https://angular.io/">Angular</a>, <a href="https://nodejs.org/">NodeJS</a> and <a href="https://mongodb.org/">MongoDB</a>.
         </p>
         <p>
-          I love to discover new places and coltures. I have been fortunate to use work as my travel excuse and so far I met the megalomaniac arabic ways of <a href="https://wikipedia.org/wiki/Dubai">Dubai, UAE</a> and the very oposit liberal city and tech shelter that is <a href="https://wikipedia.org/wiki/San_Francisco">San Francisco, CA, USA</a>.
+          I love to discover new places and cultures. I have been fortunate to use work as my travel excuse and so far I met the megalomaniac arabic ways of <a href="https://wikipedia.org/wiki/Dubai">Dubai, UAE</a> and the very opposite liberal city and tech shelter that is <a href="https://wikipedia.org/wiki/San_Francisco">San Francisco, CA, USA</a>.
```

Comment:
I made these changes because you told me too!!!! 73f760aeb87d8dff228165eceaaff5a4aeb1acb9